travel_journey/report/travel_journey_webkit.py

```diff
@@ -40,20 +40,37 @@ def _get_signer(self):
         except (KeyError, IndexError):
             return ''
 
+    # Use a class method: method that is not bound
+    # to an object,bbut to… a class!
```

Comment:
Proofreading:

```
to an object, but to… a class!
```

Your comment should explain why, not what.

We can see that a `classmethod` is used, we don't need a comment for that, but what is important to know if the reason behind that design decision.